lib/sms.js

```diff
@@ -0,0 +1,89 @@
+/* This Source Code Form is subject to the terms of the Mozilla Public
+ * License, v. 2.0. If a copy of the MPL was not distributed with this
+ * file, You can obtain one at http://mozilla.org/MPL/2.0/. */
+
+'use strict'
+
+const libpn = require('google-libphonenumber')
+const Nexmo = require('nexmo')
+const P = require('bluebird')
+const SENDER_IDS = new Map(require('./sms-sender-ids.json'))
+
+module.exports = (log, translator, templates, config) => {
+  const nexmo = new Nexmo({
+    apiKey: config.apiKey,
+    apiSecret: config.apiSecret
+  })
+  const sendSms = P.promisify(nexmo.message.sendSms, { context: nexmo.message })
```

Comment:
I don't see the point in doing it now. It's a possibility I'm aware of and that we spoke about, but writing code to handle multiple SMS APIs feels like wasted cycles at this stage. I don't foresee it being difficult to do, if/when we need it. None of the implementation/provider details leak outside of this module.